### PR TITLE
chore: update docs

### PR DESCRIPTION
**Pull request by xieshenzh**
Update docs and add release notes for 0.7.1

### CODE REVIEW

**Comment by xieshenzh** [comment]
@ritz303 please take a look at this PR which updates the docs and release notes for the coming release of the RHDA IntelliJ plugin.
The release mainly adds support for Golang and Python.

File: README.md [hunk 1]
```diff
@@ -80,17 +85,29 @@ according to your preferences.
 ### Configurable parameters
 
 **Maven** :
-<br >Path of the `mvn` executable allows Exhort to locate and execute the `mvn` commands to resolve dependencies for
+<br >Path of the `mvn` executable allows Exhort to locate and execute the `mvn` command to resolve dependencies for
```
Comment (ritz303) [inline README.md:88]: Suggest a minor rewrite here at the beginning of the sentence.

``Path of the `mvn` executable allows ...`` --> 
`Set the full path of the Maven executable, which allows ...`

File: README.md [hunk 2]
```diff
@@ -80,17 +85,29 @@ according to your preferences.
...
 Maven projects.
 Path of the `JAVA_HOME` directory is required by the `mvn` executable.
 If the paths are not provided, your IDE's `PATH` and `JAVA_HONE` environments will be used to locate the executables.
 
-**Npm** :
-<br >Path of the `npm` executable allows Exhort to locate and execute `npm` commands to resolve dependencies for Node
+**Node** :
+<br >Path of the `npm` executable allows Exhort to locate and execute the `npm` command to resolve dependencies for Node
```
Comment (ritz303) [inline README.md:94]: Suggest a minor rewrite here at the beginning of the sentence.

``Path of the `npm` executable allows ...`` --> 
`Set the full path of the Node executable, which allows ...`

File: README.md [hunk 3]
```diff
@@ -80,17 +85,29 @@ according to your preferences.
...
 projects.
 Path of the directory containing the `node` executable is required by the `npm` executable.
 If the paths are not provided, your IDE's `PATH` environment will be used to locate the executables.
 
+**Golang** :
+<br >Path of the `go` executable allows Exhort to locate and execute the `go` command to resolve dependencies for Go
```
Comment (ritz303) [inline README.md:100]: Suggest a minor rewrite here at the beginning of the sentence.

``Path of the `go` executable allows ...`` --> 
`Set the full path of the Go executable, which allows ...`

File: README.md [hunk 4]
```diff
@@ -80,17 +85,29 @@ according to your preferences.
...
+projects.
+If the path is not provided, your IDE's `PATH` environment will be used to locate the executable.
+
+**Python** :
+<br >Paths of the `python3` and `pip3` executables allow Exhort to locate and execute the `pip3` commands to resolve
```
Comment (ritz303) [inline README.md:105]: Suggest a minor rewrite here at the beginning of the sentence.

``Path of the `python3` and `pip3` executables allow ...`` --> 
``Set the full path of the Python and the package installer for Python executables, which allows Exhort to locate and execute the `python3` and `pip` commands ...``

File: README.md [hunk 5]
```diff
@@ -151,22 +169,36 @@ token [here](https://app.snyk.io/login?utm_campaign=Code-Ready-Analytics-2020&ut
   }
   ```
 
+  If you wish to ignore vulnerabilities for a dependency in a `go.mod` file, you must add `exhortignore` as a comment
+  against the dependency in the manifest file.
+  For example:
+  ```
```
Comment (ritz303) [inline README.md:175]: To add syntax highlighting, you can set the code block type after the third tick mark.  For example:

\```go

File: README.md [hunk 6]
```diff
@@ -151,22 +169,36 @@ token [here](https://app.snyk.io/login?utm_campaign=Code-Ready-Analytics-2020&ut
...
+  require (
+      golang.org/x/sys v1.6.7 // exhortignore
+  )
+  ```
+
+  If you wish to ignore vulnerabilities for a dependency in a `requirements.txt` file, you must add `exhortignore` as a
+  comment against the dependency in the manifest file.
+  For example:
+  ```
```
Comment (ritz303) [inline README.md:184]: To add syntax highlighting to this code block.

\```python

File: README.md [hunk 7]
```diff
@@ -188,6 +220,18 @@ token [here](https://app.snyk.io/login?utm_campaign=Code-Ready-Analytics-2020&ut
   }
   ```
 
+  For example, setting `exclude` attribute in the `go.mod` file:
+  ```
```
Comment (ritz303) [inline README.md:224]: To add syntax highlighting to this code block.

```go

File: README.md [hunk 8]
```diff
@@ -188,6 +220,18 @@ token [here](https://app.snyk.io/login?utm_campaign=Code-Ready-Analytics-2020&ut
...
+  exclude golang.org/x/sys v1.6.7
+
+  exclude (
+      golang.org/x/sys v1.6.7
+  )
+  ```
+
+  For example, creating an alternative file to `requirements.txt`, like `requirements-dev.txt`
+  or `requirements-test.txt` and adding the dev or test dependencies there instead.
```
Comment (ritz303) [inline README.md:233]: Suggest a rewrite of this sentence (lines 232 & 233).  For example:

``You can create an alternative file to `requirements.txt`, for example, a `requirements-dev.txt` or a `requirements-test.txt` file where you can add the development or test dependencies there.``

File: README.md (hunk 5 above)
Comment (xieshenzh) [reply README.md]: Thanks @ritz303 . I have applied the changes that you suggested.

According to the definitions in this [yaml](https://github.com/github-linguist/linguist/blob/7ca3799b8b5f1acde1dd7a8dfb7ae849d3dfb4cd/lib/linguist/languages.yml#L2486) file, keyword `text` should be used to highlight the syntax for `go.mod` file and `requirements.txt` file.

Comment (ritz303) [reply README.md]: Ah, good catch!  Thanks for making the updates, LGTM.

**Pull request merged**